### PR TITLE
Modified update_status in Generic Lifecycle to use embedded methods.

### PR DESCRIPTION
**Pull request by billfitzgerald0120**
Changed to use 2 log_and_raise and 1 log_and_notify embedded methods.
Updated spec and coverage is 100%.

### CODE REVIEW

**Comment by billfitzgerald0120** [comment]
@miq-bot add_label refactoring
@miq-bot add_reviewer @tinaafitz

screen shot 2018-10-31 at 5 40 56 pm

**Comment by billfitzgerald0120** [comment]
@mkanoor Added method instance

File: content/automate/ManageIQ/Service/Generic/StateMachines/GenericLifecycle.class/__methods__/update_status.yaml [hunk 1]
```diff
@@ -17,7 +20,7 @@ object:
       datatype: string
       priority: 1
       owner: 
-      default_value: 
+      default_value: smartvm
```
Comment (mkanoor) [inline content/automate/ManageIQ/Service/Generic/StateMachines/GenericLifecycle.class/__methods__/update_status.yaml]: @billfitzgerald0120 
Should this be smartvm?

Comment (billfitzgerald0120) [reply content/automate/ManageIQ/Service/Generic/StateMachines/GenericLifecycle.class/__methods__/update_status.yaml]: @mkanoor Not sure how this got in there but I deleted it.

**Pull request merged**